### PR TITLE
update sphinxcontrib-autoprogram URL

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by mr-c** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)
- [ ] Is the Copyright year up to date?

**Comment by mr-c** [comment]
This fixes the build error.

@luizirber @ctb Ready for review & merge

File: doc/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-http://athyra.ged.msu.edu/~mcrusoe/autoprogram/sphinxcontrib-autoprogram-0.1.1-khmerdev-20140331.tar.gz#egg=sphinxcontrib-autoprogram==0.1.1-khmerdev-20140331
+http://ci.oxli.org:8080/userContent/sphinxcontrib-autoprogram-0.1.1-khmerdev-20140331.tar.gz#egg=sphinxcontrib-autoprogram==0.1.1-khmerdev-20140331
```
Comment (luizirber) [inline doc/requirements.txt]: Any reason for the :8080? It works without the port number.

Comment (mr-c) [inline doc/requirements.txt]: See the entry for that commit (the port number is needed when downloading from ci.oxli.org itself) https://github.com/dib-lab/khmer/commit/960257b75751dd8f1e9037b3bac6e5b12ebb1601

Comment (luizirber) [inline doc/requirements.txt]: Fair enough

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**